README.md

```diff
@@ -36,6 +36,11 @@ This magic header is stripped from the request before it's forwarded to the dest
 ## Configuration
 This extension is 'plug and play' and should speak for itself. You can hover with your mouse over each field in the 'Awesome TLS' tab for more information about each field.
 
+You can either enable an additional proxy listener that will automatically apply the current fingerprint or load your custom Client Hello. You can capture it in Wireshark, copy the client hello record as a hex stream, and paste it into the field "Hex Client Hello".
+
+If you neither enable the proxy listener nor set a custom client hello hex, it will use the pre-configured fingerprints that you can select from the drop-down menu.
+![screenshot](./docs/wireshark_capture_client_hello.png)
```

Comment:
Please reduce the scale of the image and remove your mouse pointer from the screenshot. It should be be around as big as the current image, if possible (width: 805px, height: 378px).